test/index-new.html

```diff
@@ -60,7 +60,7 @@
           <br>
           <br>
 
-          <div><a id="open-app-link" class="open-app" href="javascript:;"> Open App</a></div>
+          <div><a id="open-app-link" class="open-app" href="javascript:;"> Open App - New Test</a></div>
```

Comment:
just a different _visible_ name for the button so that it's straightforward you're running the _new_ test.